tests/scm/scm-test-arity.scm

```diff
@@ -0,0 +1,43 @@
+
+(use-modules (opencog) (opencog test-runner))
+(use-modules (opencog python))
+
+(opencog-test-runner)
+(define t "opencog-arity-test")
+
+(test-begin t)
+
+(python-eval "
+from opencog.atomspace import AtomSpace, TruthValue
+from opencog.atomspace import types
+def foo(atspace):
+    TV = TruthValue(0.2, 0.69)
+    atspace.add_node(types.ConceptNode, 'Apple', TV)
+")
+
+(python-call-with-as "foo" (cog-atomspace))
+(cog-node 'ConceptNode "Apple")
```

Comment:
this line is a no-op. `(cog-node 'ConceptNode "Apple")` returns `(ConceptNode "Apple")` if it exists, else it returns `#f`.  One uses `cog-node` only on those rare occasions where you want to test to see if an atom exists already, without creating it.

Note also that an autogenerated header file defines `(ConceptNode "Apple")`  to be `(cog-new-node 'ConceptNode "Apple")`   (more or less, its slightly more complicated than that, but effectively only `cog-new-node` actually creates new nodes.)

Reply:
You might want ti say `(test-assert "atom was created" (not (eq? #f (cog-node 'ConceptNode "Apple"))))` to verify that the atom got created by the python code.